bad_code.java

```diff
@@ -0,0 +1,54 @@
+public class BadCodeExample {
+    
+    public static void main(String[] args) {
```

Comment:
It is a good practice to use braces for code blocks, even if they contain only one statement [important]